.github/workflows/ci-nym-vpn-core.yml

```diff
@@ -93,6 +93,16 @@ jobs:
           echo "TRIPLET=$triplet" >> $GITHUB_ENV
           echo "RUSTFLAGS=-L ${GITHUB_WORKSPACE}/build/lib/${triplet}" >> $GITHUB_ENV
 
+      - name: set rust flags
+        working-directory: nym-vpn-core
+        shell: bash
+        run: |
+          case  "$(uname -s)" in
```

Comment:
Looks like work in progress but I think it's probably simpler to incorporate any env variable changes in the pre-existing block?

Reply:
I don't recall adding this. @zaneschepke was it your work?